buildkite/run_check_deploy_docker_build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script checks that the image from the heroku-deploy Dockerifle
+# can be built successfully. This check is done in the buildkite-assets repo
+# to block merging/deploys to buildkite EC2 instances.
+
+DOCKER_IMAGE_NAME=check-heroku-deploy-image
+
+echo "--- Running Tests"
```

Comment:
"Tests" seems like a bit of a misnomer here, right? Are we actually running any tests? (My current understanding is that we're attempting to build a docker image.)